Mage/src/main/java/mage/abilities/effects/common/EntersBattlefieldWithXCountersEffect.java

```diff
@@ -22,16 +22,28 @@
 public class EntersBattlefieldWithXCountersEffect extends OneShotEffect {
 
     protected final Counter counter;
+    private final int multiplier;
 
     public EntersBattlefieldWithXCountersEffect(Counter counter) {
+        this(counter, 1);
+    }
+
+    public EntersBattlefieldWithXCountersEffect(Counter counter, int multiplier) {
         super(Outcome.BoostCreature);
         this.counter = counter;
-        staticText = "with X " + counter.getName() + " counters on it";
+        this.multiplier = multiplier;
+
+        String countText = "X";
+        if (multiplier == 2) {
+            countText = "twice " + countText;
+        }
```

Comment:
I was not sure what to set the text for more than 2. Any idea?

Reply:
If multiplier > 2
`CardUtil.numberToText(multiplier) + " times " + countText`